src/libs/tools/include/errors/errorTypes.hpp

```diff
@@ -0,0 +1,125 @@
+
+#ifndef ELEKTRA_ERRORTYPES_HPP
+#define ELEKTRA_ERRORTYPES_HPP
+
+#include "error.hpp"
+
+namespace kdb
+{
+namespace tools
+{
+namespace errors
+{
+
+/* Not an error by itself, but a container for multiple warnings */
```

Comment:
```suggestion
/* Not an warning by itself, but a container for multiple warnings */
```

Reply:
As I wrote in the previous message, an Error object can contain multiple Warnings so that an Error-key can directly be converted to a single C++ Error-object and to keep the behaviour consistent with the highlevel-API.
In commit c2ec7ebbb6bbddfda4f82b9d9e44aee27d9971e2 I extended the comment to explain the concept a bit more.